unit-tests/internal/CMakeLists.txt

```diff
@@ -12,11 +12,14 @@ set (INTERNAL_TESTS_SOURCES
     internal-tests-types.cpp
     internal-tests-uv-map.cpp
     internal-tests-class-logic.cpp
-    internal-tests-linux.cpp
     ../catch.h
     ../approx.h
 )
 
+if(UNIX)
```

Comment:
UNIX includes Android  as well. If it doesn't apply then it should be unix && !android